### PR TITLE
NUnit3 support

### PR DESCRIPTION
**Pull request by stevewi1950**
Added the NUnit3 task to allow NUnit3.x tests to be run using the MSBuildTasks.
Added the majority of the NUnit 3.x console command line options.
No Unit Tests created as I do not believe we will be able to reference both versions of the framework in the same test solution.

### CODE REVIEW

**Comment by niemyjski** [comment]
@pwelter34 Can we get this merged and pushed. I need support for nunit3..

File: Source/MSBuild.Community.Tasks.Tests/MSBuild.Community.Tasks.Tests.csproj [hunk 1]
```diff
@@ -69,7 +69,7 @@
     <Compile Include="AssemblyInfoTest.cs" />
     <Compile Include="AttribTest.cs" />
     <Compile Include="Beep\BeepTest.cs" />
-    <Compile Include="Checksum\ChecksumTest.cs" />
+	<Compile Include="Checksum\ChecksumTest.cs" />
```
Comment (niemyjski) [inline Source/MSBuild.Community.Tasks.Tests/MSBuild.Community.Tasks.Tests.csproj]: Can you please revert this change and the change to the readme.

**Comment by niemyjski** [comment]
@stevewi1950 Can you please merge in latest changes and revert the changes to the readme.

**Comment by stevewi1950** [comment]
Will do @niemyjski .  Will probably be sometime over the weekend at the earliest if not first thing next week as I am away from my computer at the moment.
Regards.
